alacritty/src/window.rs

```diff
@@ -281,16 +296,22 @@ impl Window {
             .with_decorations(window_config.decorations != Decorations::None)
             .with_transparent(true)
             .with_maximized(window_config.startup_mode == StartupMode::Maximized)
+            .with_min_inner_size(min_inner_size)
```

Comment:
You **always** call `set_min_inner_size` at startup anyways, why do we need `with_min_inner_size`?

Reply:
I think 1. semantically it makes sense, as we do want to have the min size set when the window is spawned, and 2. `set_min_inner_size` at startup sets to the min size calculated from the real dpi after it's determined, so if we don't have this line we will have a small time window during which there is no min size.

I think I should leave this line here but make it not set min size again if the min size calculated from the real dpi is not different from the estimated dpi.

Reply:
How does it make any difference if we set the inner size if we also request a dimension that is bigger than that?

Reply:
Wouldn't say this makes a difference in practice, as during the aforementioned time window the min size is not very important (we are not drawing on the window yet). I still think that this makes sense semantically though.